pipelines/pingcap/tiflow/latest/periodics_cdc_integration_kafka_test.groovy

```diff
@@ -0,0 +1,161 @@
+// REF: https://www.jenkins.io/doc/book/pipeline/syntax/#declarative-pipeline
+// Keep small than 400 lines: https://issues.jenkins.io/browse/JENKINS-37984
+// should triggerd for master branches
+@Library('tipipeline') _
+
+final K8S_NAMESPACE = "jenkins-tiflow"
+final GIT_FULL_REPO_NAME = 'pingcap/tiflow'
+final GIT_CREDENTIALS_ID = 'github-sre-bot-ssh'
+final POD_TEMPLATE_FILE = 'pipelines/pingcap/tiflow/latest/pod-periodics_cdc_integration_kafka_test.yaml'
+final TARGET_BRANCH = "master"
+
+pipeline {
+    agent {
+        kubernetes {
+            namespace K8S_NAMESPACE
+            yamlFile POD_TEMPLATE_FILE
+            defaultContainer 'golang'
+        }
+    }
+    environment {
+        FILE_SERVER_URL = 'http://fileserver.pingcap.net'
+    }
+    options {
+        timeout(time: 65, unit: 'MINUTES')
+        parallelsAlwaysFailFast()
+    }
+    stages {
+        stage('Debug info') {
+            steps {
+                sh label: 'Debug info', script: """
+                    printenv
+                    echo "-------------------------"
+                    go env
+                    echo "-------------------------"
+                    echo "debug command: kubectl -n ${K8S_NAMESPACE} exec -ti ${NODE_NAME} bash"
+                """
+                container(name: 'net-tool') {
+                    sh 'dig github.com'
+                }
+            }
+        }
+        stage('Checkout') {
+            options { timeout(time: 10, unit: 'MINUTES') }
+            steps {
+                dir("tiflow") {
+                    cache(path: "./", includes: '**/*', key: "git/pingcap/tiflow/rev-${TARGET_BRANCH}", restoreKeys: ['git/pingcap/tiflow/rev-']) {
+                        retry(2) {
+                            script {
+                                component.checkoutWithMergeBase('https://github.com/pingcap/tiflow.git', 'tiflow', TARGET_BRANCH, "", trunkBranch=TARGET_BRANCH, timeout=5, credentialsId="")
+                            }
+                        }
+                    }
+                }
+            }
+        }
+        stage("prepare") {
+            options { timeout(time: 20, unit: 'MINUTES') }
+            steps {
+                dir("third_party_download") {
+                    script {
+                        def tidbBranch = "master"
+                        def pdBranch = "master"
+                        def tikvBranch = "master"
+                        def tiflashBranch = "master"
+                        retry(2) {
+                            sh label: "download third_party", script: """
+                                export TIDB_BRANCH=${tidbBranch}
+                                export PD_BRANCH=${pdBranch}
+                                export TIKV_BRANCH=${tikvBranch}
+                                export TIFLASH_BRANCH=${tiflashBranch}
+                                cd ../tiflow && ./scripts/download-integration-test-binaries.sh master && ls -alh ./bin
```

Comment:
Is this script in tiflow repo?

Reply:
`third_party_download/tiflow`?
but the tiflow codes are checked out in `${WORKSPACE}/tiflow` path.

Reply:
yes